tier0/{{cookiecutter.project_slug}}/README.md

```diff
@@ -2,118 +2,10 @@
 {{ cookiecutter.project_description }}
 
 ## About the Project
-**{project statement}**
+**{project_statement}**
 
-<!--- 
```

Comment:
For sections that are recommended, instead deleting entirely, could they be kept but commented out?